WordPress/Classes/ViewRelated/NUX/SiteCreationEpilogueViewController.swift

```diff
@@ -82,6 +82,8 @@ extension SiteCreationEpilogueViewController: NUXButtonViewControllerDelegate {
         fancyAlert.modalPresentationStyle = .custom
         fancyAlert.transitioningDelegate = tabBar
         tabBar.present(fancyAlert, animated: true, completion: nil)
+
+        WPAnalytics.track(.quickStartRequestAlertViewed)
```

Comment:
Is there a way to bump this from within `FancyAlertViewController+QuickStart.swift` ?   I'm not sure. 
It would be nice tho as it would keep that control a bit more self-contained and keep some related analytics together.